plugins/out_kafka_rest/kafka.c

```diff
@@ -78,6 +78,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_kafka_rest, tag_key),
      "Specify the key name of the record if include_tag_key is enabled. "
     },
+    {
+     FLB_CONFIG_MAP_STR, "avro", false,
```

Comment:
Could you modify these point ?

1. `FLB_CONFIG_MAP_STR` -> `FLB_CONFIG_MAP_BOOL`
2. false -> "false"

Below configuration is helpful.
```
    {
     FLB_CONFIG_MAP_BOOL, "include_tag_key", "false",
     0, FLB_TRUE, offsetof(struct flb_kafka_rest, include_tag_key),
     "Specify whether to append tag name to final record. "
    },
```

Reply:
Could you rename the name of property `avro` ?
User may not imagine how it works.  The property is to modify http header.
e.g. `avro_http_header` or something.

Reply:
Thank you for updating.

`FLB_CONFIG_MAP_BOOL, "avro_http_header", false,`
3rd parameter should be string type.
So it should be "false".